customer-commons/src/main/java/no/unit/nva/customer/service/impl/DynamoDBCustomerService.java

```diff
@@ -141,6 +143,19 @@ public void createChannelClaim(UUID customerIdentifier, ChannelClaimDto channelC
         putCustomer(customerIdentifier, customer.addChannelClaim(channelClaim), false);
     }
 
+    @Override
+    public Collection<ChannelClaimWithClaimer> getChannelClaims() {
+        return getCustomers().stream()
+                   .flatMap(customer -> customer.getChannelClaims().stream()
+                                            .map(claim -> getChannelClaimWithClaimer(claim, customer)))
+                   .collect(Collectors.toList());
```

Comment:
Are we on <java21? if not, simply toList. Fair enough to say this method returns `List<ChannelClaimWithClaimer>` since it what is being returned. Postel's law "be conservative in what you do, be liberal in what you accept from others", I guess.

Reply:
Here is java 17, yes :)